pepper-apis/studybuilder-cli/studies/singular/patches/ddp-8554-self-validation.conf

```diff
@@ -0,0 +1,38 @@
+{
+  "validations": [
+
+    # When using the Enroll Myself flow, if a user enters an age at or above the AOM for their region in
+    # the pre-qual but enters a date of birth that equates to <AOM for their region in the adult self consent,
+    # an error message should appear under the consent DOB field.
+    # Formally: CONSENT.DOB < AOM
+    {
+      "messageTemplate": {
+        "templateType": "HTML",
+        "templateText": "$singular_consent_self_validation",
+        "variables": [
+          {
+            "name": "singular_consent_self_validation",
+            "translations": [
+              {
+                "language": "en",
+                "text": """Please check the date of birth you have entered. If the date is correct, then you are
+                               younger than the age required by local law to self-enroll in this study. However, you
+                               may ask your parent or guardian to enroll you in the study. If you think there is a
+                               mistake, please reach out to us at (650) 561-6750 or
+                               <a href="mailto:contact@projectsingular.org">contact@projectsingular.org</a>.
+                            """
+              }
+            ]
+          }
+        ]
+      },
+      "stableIds": ["CONSENT_SELF_DATE_OF_BIRTH"],
+      "precondition": """
+                          user.studies["singular"].forms["CONSENT_SELF"].questions["CONSENT_SELF_DATE_OF_BIRTH"].isAnswered()
+                          """,
+      "expression": """
+                        user.studies["singular"].forms["CONSENT_SELF"].questions["CONSENT_SELF_DATE_OF_BIRTH"].answers.ageAtMost(18, YEARS)
```

Comment:
Since the user is self and already entered AGE and location in PREQUAL, we could do AOM rules by location check rather than just 18 years. 
If we are ok with checking 18 years, this is cleaner and simple though.

Reply:
We discussed it with Marco and Whitney and decided to simplify comparison to 18

Reply:
Anyway, I changed format criteria